electron-builder.config.js

```diff
@@ -24,6 +25,19 @@ const isMac = process.platform === "darwin";
 // cf: https://k-hyoda.hatenablog.com/entry/2021/10/23/000349#%E8%BF%BD%E5%8A%A0%E5%B1%95%E9%96%8B%E3%83%95%E3%82%A1%E3%82%A4%E3%83%AB%E5%85%88%E3%81%AE%E8%A8%AD%E5%AE%9A
 const extraFilePrefix = isMac ? "MacOS/" : "";
 
+const sevenZipFile = fs
+  .readdirSync(path.resolve(__dirname, "build", "vendored", "7z"))
+  .find(
+    // 拡張子なし（Linux/Macバイナリ）と7za.exeのみを埋め込み
+    (fileName) => !fileName.includes(".") || fileName === "7za.exe"
```

Comment:
この判定だとREADME.md辺りの拡張子を消しただけでバグっちゃうので、まあwindowsのときはファイル名指定してるし、全部網羅しちゃうのはどうでしょう。
```suggestion
    // 拡張子なし（Linux/Macバイナリ）と7za.exeのみを埋め込み
    (fileName) => fileName == "7za.exe" || fileName == "7zzs" || fileName == "7zz"
```